[2023-06-25 00:08:11,898] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-24T00:00:00+00:00 [queued]>
[2023-06-25 00:08:12,005] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-24T00:00:00+00:00 [queued]>
[2023-06-25 00:08:12,009] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-06-25 00:08:12,013] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2023-06-25 00:08:12,015] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-06-25 00:08:12,103] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_zero> on 2023-06-24T00:00:00+00:00
[2023-06-25 00:08:12,122] {standard_task_runner.py:52} INFO - Started process 26703 to run task
[2023-06-25 00:08:12,145] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_zero', '2023-06-24T00:00:00+00:00', '--job-id', '41', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpv3yujgd9', '--error-file', '/tmp/tmpcphisg7l']
[2023-06-25 00:08:12,151] {standard_task_runner.py:77} INFO - Job 41: Subtask task_zero
[2023-06-25 00:08:12,435] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_zero 2023-06-24T00:00:00+00:00 [running]> on host 7f7a4865b07a
[2023-06-25 00:08:12,755] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_zero
AIRFLOW_CTX_EXECUTION_DATE=2023-06-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-24T00:00:00+00:00
[2023-06-25 00:08:12,767] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-06-25 00:08:12,772] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pip install beautifulsoup4']
[2023-06-25 00:08:12,812] {subprocess.py:75} INFO - Output:
[2023-06-25 00:08:30,616] {subprocess.py:79} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-06-25 00:08:32,056] {subprocess.py:79} INFO - Requirement already satisfied: beautifulsoup4 in /home/***/.local/lib/python3.6/site-packages (4.12.2)
[2023-06-25 00:08:32,124] {subprocess.py:79} INFO - Requirement already satisfied: soupsieve>1.2 in /home/***/.local/lib/python3.6/site-packages (from beautifulsoup4) (2.3.2.post1)
[2023-06-25 00:10:08,822] {subprocess.py:83} INFO - Command exited with return code 0
[2023-06-25 00:10:08,936] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_zero, execution_date=20230624T000000, start_date=20230625T000811, end_date=20230625T001008
[2023-06-25 00:10:09,069] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-25 00:10:09,115] {local_task_job.py:151} INFO - Task exited with return code 0
